tests/integration/test_tenants.py

```diff
@@ -30,12 +31,30 @@ def test_tenants_POST(client, auth_headers):
         "staffIDs": [1, 2]
     }
 
+    newTenantWithLease = {
+        "firstName": "Finn",
+        "lastName": "The Human",
+        "phone": "123-555-4321",
+        "propertyID": 2,
+        "occupants": 3,
+        "dateTimeEnd": Time.one_year_from_now(),
+        "dateTimeStart": Time.yesterday(),
```

Comment:
The frontend sends us an iso8601 time format. Can we test with that time format. I believe this test would fail with that time format, because the schema expects the same format that the Time class returns. So we'll need to update the `dateTimeEnd` and `dateTimeStart` fields in the schema to work with the iso8601 format. 

It would be fine to hard-code dates here if Python doesn't have anything we can use to get an iso8601 time format.

Reply:
We could update the schema for the lease so that it will accept the iso format. That can be done here: https://github.com/codeforpdx/dwellinglybackend/blob/development/schemas/lease.py#L14